_posts/2015-05-22-about-the-quality-and-fun.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: About the quality and fun!
+author: ruoat
+excerpt: Software quality is important. But let's not forget the fun!
+---
+
+About the quality
+-----------------
+
+Two weeks ago there was annual Solita Codecamp where developer-oriented people can try out and play with not so enterprisey software than we do usually every day.
```

Comment:
'not' pelaa yhteen 'as'-sanan kanssa:
not so ... as we
(vs. less enterprisey than...)